src/game/game.c

```diff
@@ -271,49 +290,182 @@ const CameraMain gUnknown_080D5A10[] = {
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x40, 0x02, 0x1C }, //
+    { 0x20, 0x40, 0x02, 0x1C }, //
+    { 0x20, 0x40, 0x02, 0x1C }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x40, 0x02, 0x1C }, //
+    { 0x20, 0x40, 0x02, 0x1C }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x40, 0x02, 0x1C }, //
+    { 0x40, 0x20, 0x02, 0x1C }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1C }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1C }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x40, 0x02, 0x1C }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x40, 0x20, 0x02, 0x1C }, //
+};
 
-asm("    .global gUnknown_080D5B22\n"
-    "gUnknown_080D5B22:\n"
-    "    .incbin \"baserom.gba\", 0x000D5B22, 0x2E");
+const u8 gUnknown_080D5B20[] = {
+    14,  0, 1, //
+    22,  0, 3, //
+    30,  0, 2, //
+    38,  0, 1, //
+    46,  0, 3, //
+    62,  0, 1, //
+    70,  0, 2, //
+    86,  0, 1, //
+    94,  0, 2, //
+    126, 0, 1, //
+    168, 1, 1, //
+    174, 1, 2, //
+    182, 2, 3, //
+    198, 3, 4, //
+    222, 4, 5, //
+    255, 5, 6, //
+};
 
-asm("    .global gUnknown_080D5B50\n"
-    "gUnknown_080D5B50:\n"
-    "    .incbin \"baserom.gba\", 0x000D5B50, 0xA0\n");
+// Used by sub_801E498
+const u8 gUnknown_080D5B50[160] = {
+    // 0-105
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 10, //
+    10, 10, 8, //
+    8, 8, 8, //
+    7, 7, 7, //
+    7, 6, 6, //
+
+    // 106-159 | This data appears to be unused
+    6, 6, 6, //
+    6, 6, 6, //
+    5, 5, 5, //
+    5, 5, 5, //
+    5, 5, 4, //
+    4, 4, 4, //
+    4, 4, 4, //
+    4, 3, 3, //
+    3, 3, 3, //
+    3, 3, 3, //
+    3, 3, 3, //
+    3, 3, 3, //
+    3, 3, 2, //
+    2, 2, 2, //
+    2, 2, 2, //
+    2, 2, 2, //
+    2, 2, 2, //
+    2, 2, 2, //
+
+    // Maybe padding?
+    0
```

Comment:
Probably. Are you sure it's needed? The next array should automatically add the padding.